### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by qkrybin**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [ ] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [x] others (anything other than above)

---

### 2. Summary

**Related:** https://github.com/dsrvlabs/vatz/issues/309

**Summary**
update configuring default.yaml and starting process 

---

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -33,17 +33,94 @@
...
-    - Port
-        - Check your machine used port number. If you set it to a port number that is already in use, an error will occur.
-    - discord_secret
-        - If you want to receive discord messages, register webhook URL.
 
-- Execute each binary as below <br> <img width="1510" alt="image" src="https://user-images.githubusercontent.com/106724973/180962626-c4de859c-526d-4038-87b8-badebed44136.png">
-- Alert notification will be sent if block height is not increased <br> <img width="223" alt="image" src="https://user-images.githubusercontent.com/106724973/180962941-4db55b66-ceb6-4cd4-bd53-96d21bdf5596.png">
+### Plugin Infos
+> **You can find further plugins here.**
+> - [vatz-plugin-common](https://github.com/dsrvlabs/vatz-plugin-common)
```
Comment (xellos00) [inline docs/installation.md:89]: @qkrybin @rootwarp 
I don't think `vatz-plugin-common` is our official plugin, do we need to put this on here?
what do you guys think? 

I hope not to put here unless we keep manage the plugin.

File: docs/installation.md [hunk 2]
```diff
@@ -33,17 +33,94 @@
...
+> - [vztz-plugin-cosmoshub](https://github.com/dsrvlabs/vatz-plugin-cosmoshub)
+
+1. plugin_name
+   - refer to `vatz-plugin-common/plugins/cosmos-sdk-blocksync/main.go` of official repository
+      - Search `pluginName`in vat-plugin and update the default.yaml with `pluginName`
+2. plugin_port
+   - set your plugin used port number to connect the plugin.
+
+
+## 2. Execute each binary
```
Comment (xellos00) [inline docs/installation.md:100]: @qkrybin 
Can you add installation of VATZ plugins with CLI on this?
or 
update document late with all CLI usage?
which one do you prefer?

File: docs/installation.md (hunk 1 above)
Comment (rootwarp) [reply docs/installation.md:89]: @xellos00 Oh, you're right.
For this document, I think only including the official plugins will be good.

Comment (xellos00) [reply docs/installation.md:89]: @qkrybin 
Can you get rid of vatz-plugin-common from installation.md?

Comment (qkrybin) [reply docs/installation.md]: I removed it!

File: docs/installation.md (hunk 2 above)
Comment (qkrybin) [reply docs/installation.md:99]: oh i didn't know there is other ways to install the vatz plugins. I can update cli plugin intall here. And I will organize the full CLI usage later.

**Review by rootwarp — APPROVED** [review]

File: docs/installation.md (hunk 2 above)
Comment (qkrybin) [reply docs/installation.md:99]: @xellos00 
hm I'm trying to add 'vatz plugin install' but how can I found weather or not it's successfully installed without 'plugin list cmd'?? And If it need to be changed agian after all plugin cmd is added, I think it's better to update them at a time.

Comment (xellos00) [reply docs/installation.md:99]: @qkrybin 
Agree!!, I will ping you when basic command is fully developed.

**Review by xellos00 — APPROVED** [review]

**Pull request merged**